src/style.css

```diff
@@ -1 +1,71 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(../src/fonts/Avenir-Book.ttf), url(../src/fonts/Avenir-Heavy.ttf);
```

Comment:
The `@font-face` rule should include a format hint for each font file to improve compatibility and performance. For example, `src: url('../fonts/Avenir-Book.ttf') format('truetype'), url('../fonts/Avenir-Heavy.ttf') format('truetype');`